src/app/products/recurring-deposit-products/view-recurring-deposit-product/view-recurring-deposit-product.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
The dailog component has been removed and the screen has been updated with the Incentives table. Please have a look at the GIF attached in the description to have a look at the UI implemented. Let me know if any further changes required.

Reply:
No, don't create a separate component for the interest rate chart. Follow the approach as in the Preview step of Fixed Deposit Component. (when New Fixed Deposit Product is created in the web-app)

Reply:
@abhaychawla As much as I have understood, should I implement the edit option on clicking the `Edit Button` for the interest rate chart in the same page(view recurring deposit product page) instead of navigating to the new page. The table will be hidden and the slabs will be displayed as editable forms and also an option to add a new slab will be available.

Reply:
For the time being, we will not have this separate edit option now for interest rate chart while viewing the product, we will only have it during the editing of the product. So no need for that. Just like the preview screen of Create Fixed Deposit Product make sure the interest rate chart slabs and incentives are displayed in table here without any option for editing them.

Reply:
I got it. I will update it.